config/application.rb

```diff
@@ -122,19 +141,15 @@ class Application < Rails::Application
     #   https://bugs.ruby-lang.org/issues/14372
     #
 
-    # TODO: Remove this once we move to config.load_defaults 7.0 as this is the default.
-    # Note, rails 7 can read cache format from 6 or 7 so there is no risk if you're running rails 7.
-    # See: https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#new-activesupport-cache-serialization-format
-    warn "Warning: Remove redundant config.active_support.cache_format_version = 7.0 from #{__FILE__}:#{__LINE__ + 1} if using config.load_defaults 7.0" if config.active_support.cache_format_version == 7.0
-    config.active_support.cache_format_version = 7.0
-
```

Comment:
From the the commit message:

Note, cache_format_version is backward compatible:
New versions can read old versions.  During rolling deployments
old versions would not be able to read new versions so you must
upgrade code first, then, they'll be able to read old or new versions.

For us, we always deploy code first and restart so changing formats to
the latest will not be a problem.

See also: https://www.github.com/rails/rails/pull/48122